src/sql-parser/src/parser.rs

```diff
@@ -2044,6 +2044,10 @@ impl<'a> Parser<'a> {
                 self.expect_keyword(FULLNAME)?;
                 name
             }
+            NULL => {
+                self.expect_keyword(DEFAULTS)?;
```

Comment:
Add a test for this new syntax, probably in https://github.com/MaterializeInc/materialize/blob/main/src/sql-parser/tests/testdata/ddl

Reply:
Cool, will add those

Reply:
added!